src/huggingface_hub/keras_mixin.py

```diff
@@ -144,6 +151,172 @@ def _create_model_card(
+                - `"epoch"`: Save is done at the end of each epoch.
+                - `"steps"`: Save is done every `save_steps`
+        save_steps (`int`, *optional*):
+            The number of steps between saves when using the "steps" `save_strategy`.
+        hub_token (`str`, *optional*):
+            The token to use to push the model to the Hub. Will default to the token in the cache folder obtained with
+            `huggingface-cli login`.
+                api_endpoint (:obj:`str`, `optional`):
+            The API endpoint to use when pushing the model to the hub.
+        organization (:obj:`str`, `optional`):
+            Organization in which you want to push your model or tokenizer (you must be a member of this
+            organization).
+        api_endpoint (:obj:`str`, `optional`):
+            The API endpoint to use when pushing the model to the hub.
+        git_user (``str``, `optional`):
+            will override the ``git config user.name`` for committing and pushing files to the hub.
+        git_email (``str``, `optional`):
+            will override the ``git config user.email`` for committing and pushing files to the hub.
+        task_name (:obj:`str`, `optional`):
+            Name of the task the model was trained on. See the available tasks at https://github.com/huggingface/huggingface_hub/blob/main/js/src/lib/interfaces/Types.ts.
+        model_plot (:obj:`bool`):
+            Setting this to `True` will plot the model and put it in the model card. Requires graphviz and pydot to be installed.
+        include_optimizer (:obj:`bool`, `optional`):
+            Whether or not to include optimizer during serialization.
+        model_save_kwargs(:obj:`dict`, `optional`):
+            model_save_kwargs will be passed to tf.keras.models.save_model().
+    """
+
+    def __init__(
+        self,
+        save_strategy: Optional[str] = "epoch",
+        save_steps: Optional[int] = None,
+        repo_path_or_name: Optional[str] = None,
+        repo_url: Optional[str] = None,
+        hub_token: Optional[str] = None,
+        organization: Optional[str] = None,
+        api_endpoint: Optional[str] = None,
+        git_user: Optional[str] = None,
+        git_email: Optional[str] = None,
+        model_plot: Optional[bool] = True,
+        include_optimizer: Optional[bool] = True,
+        task_name: Optional[str] = None,
+    ):
+        super().__init__()
+        self.save_strategy = save_strategy
+        self.save_steps = save_steps
+        self.repo_path_or_name = repo_path_or_name
+        self.repo_url = repo_url
+        self.organization = organization
+        self.hub_token = hub_token
+        self.api_endpoint = api_endpoint
+        self.model_plot = model_plot
+        self.task_name = task_name
+        self.last_job = None
+        self.git_user = git_user
+        self.git_email = git_email
+        self.include_optimizer = include_optimizer
+        self.task_name = task_name
```

Comment:
* Please make sure the docstring, the params, and attribute assignment are aligned
* Some of this params are just used in the `__init__`, so it does not give any value to assign them as attributes. Some examples are `repo_url`, `repo_path_or_name`, git user and others